nle/nethack/nethack.py

```diff
@@ -115,7 +115,7 @@ def __init__(
         shutil.copyfile(DLPATH, dlpath)
 
         if options is None:
-            options = NETHACKOPTIONS
+            options = list(NETHACKOPTIONS)
```

Comment:
Is this `list` call necessary given the next line?

Reply:
You're right!